frontend/packages/integration-tests-cypress/views/overview.ts

```diff
@@ -0,0 +1,14 @@
+export const overviewPage = {
+  projectOverviewIsVisible: () => {
+    cy.get(`.odc-topology-list-view`).should('be.visible');
+  },
+  itemsAreVisible: () => {
```

Comment:
Recommend `itemsShouldBeVisible` for consistency

Reply:
Done